jbpm-task-assigning/kie-server-services-task-assigning/kie-server-services-task-assigning-user-system/kie-server-services-task-assigning-user-system-simple/src/main/java/org/kie/server/services/taskassigning/user/system/simple/WildflyUtil.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.services.taskassigning.user.system.simple;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.commons.io.IOUtils;
+import org.kie.server.services.taskassigning.user.system.api.Group;
+import org.kie.server.services.taskassigning.user.system.api.User;
+
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+public class WildflyUtil {
```

Comment:
Did we ever consider what happens on Websphere, Weblogic and Tomcat?
(I'm not sure we're being asked to deliver those. Traditionally, we would have been. A question for PM, perhaps?)

Reply:
I'm still pending to send the document with the configuration options, etc. Will have today.

But the name WildlfyUtil in this context means that the "UserSystemServiceSimple" implementation is a "simple" one, that reads the users definitions from a file with the format
userA=group1,group2
userB=group1,group3
and so on, it's a kind of "development" mode user system service implementation.
But is not tight to any particular container.
Deployments in other containers than WF can still use this format for "development", "testing" or even production (this last option is less likely). The only thing they have to do is to copy the configuration file in a directory that can be accessed by kie-server webapp  and configure the "UserSystemServiceSimple" with the parameter
        <property name="org.kie.server.services.taskassigning.user.system.simple.users" value="file:///XXXXXX/roles.properties"/>

As long the kie-server webapp can access the directory XXXXX it'll work.